app/utils/query_helpers.py

```diff
@@ -19,37 +19,54 @@
     DeadlockedTrainingStats
 )
 
+from app.models.uya import (
+    UyaPlayer,
+    UyaOverallStats,
+    UyaWeaponStats,
+    UyaSiegeStats,
+    UyaCTFStats
+)
+
 from app.schemas.schemas import StatOffering
 from horizon.parsing.deadlocked_stats import vanilla_stats_map, custom_stats_map
+from horizon.parsing.uya_stats import uya_vanilla_stats_map
 
 
-def update_deadlocked_player_vanilla_stats(
+def update_player_vanilla_stats(
+    game: str,
     session: Session,
     player_id: int | str,
     player_name: str,
     wide_stats: list[int]
 ) -> None:
```

Comment:
I'm probably going to compress the vanilla and custom stats down to a single function.